### PR TITLE
Implement WorkflowCardBody Component

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 5099 .

#### Changes proposed in this Pull Request:

- [X] - Update JsDocs

- [X] - Write test cases with 80% coverage

- [X] - Update the demo in an example project to showcase the component.

- [X] - Update Readme

- [X] - Test in Dark / Light Theme

- [X] - Test It in RTL view

- [X] - Styling as per https://brightlayer-ui.github.io/patterns/forms#mobile

#### Screenshots / Screen Recording (if applicable)

- 
Screenshot 2024-01-18 at 12 32 08 PM

Screenshot 2024-01-23 at 5 53 48 PM

Screenshot 2024-01-23 at 5 56 27 PM

#### To Test:

- Go to the homepage click on workflowCardBody Example button

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by JeffGreiner-eaton** [comment]
This looks good on Android phone, unable to verify the hook & styles for tablet if 
- someone else would please verify that and I will approve this PR

Also we need to figure out if we want to continue to use react-test-renderer or just use { render } from '@testing-library/react-native'

This would remove the setup stuff thats required from the renderer.
- From this
```
describe('WorkflowCardBody Test', () => {
    afterEach(cleanup);
    it('WorkflowCardBody renders correctly', () => {
        const rendered = renderer
            .create(
                <WorkflowCardBody>
                    <Text>This is workflow card body content.</Text>
                </WorkflowCardBody>
            )
            .toJSON();
        expect(rendered).toBeTruthy();
    });
});
```
- To this

```
describe('WorkflowCardBody Test', () => {
    afterEach(cleanup);
    it('WorkflowCardBody renders correctly', () => {
        render(
                <WorkflowCardBody>
                    <Text>This is workflow card body content.</Text>
                </WorkflowCardBody>
        )
            .toJSON();
        expect(render).toBeTruthy();
    });
});
```
we also need to remove anything with enzyme

**Comment by JeffGreiner-eaton** [comment]
refer to my comment above, was able to verify tablet iPad working with style

image
image

File: login-workflow/src/__tests__/components/WorkflowCard/WorkflowCardBody-test.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
```
Comment (JeffGreiner-eaton) [inline login-workflow/src/__tests__/components/WorkflowCard/WorkflowCardBody-test.tsx:1]: maybe something like this..

/**
 * @format
 */

import React from 'react';
import { cleanup, render, screen } from '@testing-library/react-native';
// import renderer from 'react-test-renderer';
import { WorkflowCardBody } from 'src/components/WorkflowCard';
import { Text } from 'react-native-paper';

describe('WorkflowCardBody Test', () => {
    afterEach(cleanup);
    it('WorkflowCardBody renders correctly', () => {
        render(
                <WorkflowCardBody>
                    <Text>This is workflow card body content.</Text>
                </WorkflowCardBody>
        )
            .toJSON();
        expect(render).toBeTruthy();
        expect(screen.getByText('This is workflow card body content.')).toBeTruthy();
    });
});

**Review by surajeaton — APPROVED** [review]
Simulator Screenshot - iphone 12 pro - 2024-01-24 at 12 28 37
Looks good and works fine on ios.

Just the small changes @JeffGreiner-eaton suggested.

**Pull request merged**